.github/workflows/build-image.yml

```
@@ -0,0 +1,72 @@
+name: Re-build and publish key4hep-externals image
+on: [push, workflow_dispatch]
```

```suggestion
on:
  push:
    branches:
    - main
  workflow_dispatch:
```
Probably we don't want to run every time there is a push to any branch. Like this I think it would only work after there is a push either to `main` directly or through a PR.